fda/grid.py

```diff
@@ -44,6 +45,242 @@ def _list_of_arrays(original_array):
     return list(new_array)
 
 
+class _GridInterpolator:
```

Comment:
If this interpolator is only for splines, maybe it should say so in its name.

Reply:
Changed in 8c58331.